src/components/plots/helpers/changeEmbeddingAxesIfNecessary.js

```diff
@@ -0,0 +1,15 @@
+const changeEmbeddingAxesIfNecessary = (config, method, onUpdate) => {
```

Comment:
I would name this `renameEmbeddingAxes`

Reply:
but we also check so they don't change every time

Reply:
that is true. It is just a suggestion, you can keep the name as it is